bpmn-model/src/main/java/io/zeebe/model/bpmn/validation/zeebe/ZeebeCalledElementValidator.java

```diff
@@ -30,13 +37,21 @@
   @Override
   public void validate(
       final ZeebeCalledElement element, final ValidationResultCollector validationResultCollector) {
-    final String processId = element.getProcessId();
 
-    if (processId == null || element.getProcessId().isEmpty()) {
-      validationResultCollector.addError(
-          0,
-          String.format(
-              "Attribute '%s' must be present and not empty", ZeebeConstants.ATTRIBUTE_PROCESS_ID));
+    final Optional<String> processId =
+        Optional.ofNullable(element.getProcessId()).filter(notEmpty());
```

Comment:
```suggestion
        Optional.ofNullable(element.getProcessId()).filter(Predicate.not(String::isEmpty));
```

This way we can remove the `notEmpty` method

Reply:
Unfortunately, `Predicate.not()` is not available in Java 8 :sweat_smile: I was added with Java 11. So, we can't use it in the model API.

Reply:
Ah I see, I thought we were only using Java 8 in the client

Reply:
Correct but the model API is also used by the client :sweat_smile: